flaskr/models/models.py

```diff
@@ -103,28 +106,37 @@ def __repr__(self):
 
 class Assignment:
 
-    def __init__(self, assignment_id, subject_id, assignment_name):
+    def __init__(self, assignment_id, subject_id, assignment_name, due_date_time=None):
         self.id = assignment_id
         self.subject_id = subject_id
         self.name = assignment_name
-
+        self.due_date_time = due_date_time
+        if due_date_time:
+            # Seperates date_time into just date and time
+            self.due_date = due_date_time[:10]
```

Comment:
This will cause lots of errors if number of chars change.
There is a library in python to handle date & time separation.